### DISCUSSION TITLE
history parser

### DISCUSSION THREAD

**Discussion by slippers** [Wikilicious/pygmc#125] [General]
here is a bit of code i left out of gmc [history.py](https://gitlab.com/-/snippets/3733239).  

had a lot of fun building it.  just ran out of time to push it along.

**Comment by Wikilicious** [comment]
Hi,
Thanks for sharing. I still need to get my hands on a source to push the counts into 4bytes to learn how that gets recorded for the history parser. 

I have come across your project and listed it in my readme... I wasn't sure what URL to use... Is it ok for me to link to https://gitlab.com/slippers/gmc ?
